### PR TITLE
Release v1.2.2

### PR DESCRIPTION
**Pull request by jathanism**
Request to merge `develop` into `main` for release v1.2.2.

### CODE REVIEW

**Comment by glennmatthews** [comment]
Let's aim to get the following included in this release as well:

- [ ] #1181 
- [ ] #1188 
- [ ] #1200 
- [ ] #1205

File: nautobot/docs/release-notes/version-1.2.md [hunk 1]
```diff
@@ -140,6 +140,18 @@ All models that have `slug` fields now use `AutoSlugField` from the `django-exte
 
 Just as with the UI, the `slug` can still always be explicitly set if desired.
 
+## v1.2.2 (TBD)
+### Added
+
+- [#1152](https://github.com/nautobot/nautobot/pull/1152) - Added Add REST API and GraphQL for `JobLogEntry` objects.
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.2.md:146]:
```suggestion
- [#1152](https://github.com/nautobot/nautobot/pull/1152) - Added REST API and GraphQL for `JobLogEntry` objects.
```

File: nautobot/docs/release-notes/version-1.2.md [hunk 2]
```diff
@@ -140,6 +140,18 @@ All models that have `slug` fields now use `AutoSlugField` from the `django-exte
 
 Just as with the UI, the `slug` can still always be explicitly set if desired.
 
+## v1.2.2 (TBD)
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.2.md:143]:
```suggestion
## v1.2.2 (2021-12-27)
```

File: pyproject.toml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "1.2.1"
+version = "1.2.2-beta.1"
```
Comment (glennmatthews) [inline pyproject.toml:5]:
```suggestion
version = "1.2.2"
```

**Review by glennmatthews — APPROVED** [review]

**Review by ubajze — APPROVED** [review]

File: nautobot/extras/api/views.py [hunk 4]
```diff
@@ -439,6 +450,13 @@ class JobResultViewSet(ModelViewSet):
     serializer_class = serializers.JobResultSerializer
     filterset_class = filters.JobResultFilterSet
 
+    @action(detail=True)
+    def logs(self, request, pk=None):
+        job_result = self.get_object()
+        logs = JobLogEntry.objects.filter(job_result=job_result)
```
Comment (jathanism) [inline nautobot/extras/api/views.py:456]:
```suggestion
        logs = job_result.logs.all()
```

**Pull request merged**